genon/preprocessor/facade/parser_processor.py

```
@@ -2015,9 +2034,30 @@ def _parse_audio(self, file_path: str, **kwargs) -> str:
+        """
+        from genon.preprocessor.converters.xlsx_processor import load_tables
+
+        tables = load_tables(
+            file_path,
+            header_row=self._xlsx_cfg["header_row"],
+            multi_table=self._xlsx_cfg["multi_table"],
+        )
+        data: list[dict] = []
+        for t in tables:
+            headers = t["headers"]
+            data_rows = [dict(zip(headers, values)) for values in t["data_rows"]]
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)